### PR TITLE
Switching worth system to an automatic value generation proc.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
DNM, WIP

Basically just making items generate their own worth when examined based on the actual use and purpose of the item.

Kills `worths_list.dm`.

### CODE REVIEW

**Comment by MistakeNot4892** [comment]
Numbers seem to be closing in on more or less reasonable for now (except for guns and weapons). Will work out some proc to dump the entire worth list to CSV or something so we can discuss it another day.

File: code/game/machinery/_machines_base/machinery.dm [hunk 1]
```diff
@@ -457,4 +457,9 @@ Class Procs:
 		var/obj/item/stock_parts/part = get_component_of_type(path)
 		var/list/part_costs = part.building_cost()
 		for(var/key in part_costs)
-			.[key] += part_costs[key] * component_types[path]
\ No newline at end of file
+			.[key] += part_costs[key] * component_types[path]
+
+/obj/machinery/get_monetary_worth()
```
Comment (comma) [inline code/game/machinery/_machines_base/machinery.dm]: maybe count value of all components?

**Comment by afterthought2** [comment]
Arguably it can even iterate contents and add in the worth of everything there. Not entirely sure that's reasonable, though.

**Comment by MistakeNot4892** [comment]
It already does that.

**Review by afterthought2 — APPROVED** [review]
Read through it; looks reasonable. Machinery is already counting components.

**Pull request merged**